core/src/main/java/hera/core/api/handlers/YouTubeApiHandler.java

```diff
@@ -46,10 +46,10 @@ public static void initialise() {
 
 	public static String getYoutubeVideoFromKeyword(String keyword) {
 		try {
-			YouTube.Search.List search = youtube.search().list(Collections.singletonList("id,snippet"));
+			YouTube.Search.List search = youtube.search().list("id,snippet");
 			search.setKey(apiToken);
 			search.setQ(keyword);
-			search.setType(Collections.singletonList("video"));
+			search.setType("video");
 			search.setFields("items(id/kind,id/videoId,snippet/title,snippet/thumbnails/default/url)");
```

Comment:
these two lines suddenly caused an error so i changed them to make it stop tell me. not tested might break something will look into later